test/gui/tst_syncing/test.feature

```diff
@@ -31,7 +31,7 @@ Feature: Syncing files
         And the folder "simple-folder" should exist on the file system
         And the folder "large-folder" should exist on the file system
 
-    @issue-9733
+    @issue-9733 @skip
```

Comment:
I will work on separate PR for these skipped tests which are causing a bit of flakiness in the CI

Reply:
issue https://github.com/owncloud/client/issues/11619 occurs more in these scenarios